README.md

```diff
@@ -1,12 +1,12 @@
 # Kadena.ex
 
-**Kadena.ex** is an Open-Source library for Elixir that allows developers to interact with Kadena and its Chainweb.
+**Kadena.ex** is an Open-Source library for Elixir that allows developers to interact with Kadena, Chainweb, and PACT.
```

Comment:
@miguelnietoa my suggestion: `**Kadena.ex** is an open source library for Elixir that allows developers to interact with the Kadena Chainweb.`